dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -750,7 +751,8 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 								</Hide>
 
 								{showTopicFilterBank &&
-									CAPIArticle.availableTopics && (
+									CAPIArticle.availableTopics &&
```

Comment:
if I'm understanding correctly then is looks like this set of 3 conditions is equivalent to `!showToggle`. If that's the intention, there might be scope to rename `showToggle` and then use it here as well as where it's used above.

Just a suggestion if I've understood correctly, and if it seems like it'll help to make the logic easier to follow. Not a blocking suggestion for this PR  though!